recipe/set_bld_opts.bat

```diff
@@ -17,6 +17,8 @@ FOR /F "tokens=1,2 delims=." %%a IN ("%poppler%") DO (
   set POPPLER_MAJOR_VERSION=%%a
   set POPPLER_MINOR_VERSION=%%b
 )
+:: remove leading zeros
+set /a POPPLER_MINOR_VERSION=10000%POPPLER_MINOR_VERSION% %% 10000
```

Comment:
@gillins I also wondered if there was any value in stealing stuff from the the [GDAL Windows build yml file](https://github.com/OSGeo/gdal/blob/0f7213e75417a02470a4675f4486255dd8deb21d/.github/workflows/windows_build.yml#L171-L175). I'm far from an expert on compiling stuff so not sure if this is helpful or not but figured I'd put it out just in case :)